packages/tuono/src/build/index.ts

```diff
@@ -65,6 +74,7 @@ const developmentSSRBundle = (): void => {
       mergeConfig<InlineConfig, InlineConfig>(
         createBaseViteConfigFromTuonoConfig(config),
         {
+          plugins: VITE_SSR_PLUGINS,
```

Comment:
Edge case: what if user specify an inject plugin with the same polypill?

Reply:
Well, why the user should ever do that? We basically provide that out of the box to them. I'd consider ignoring this for now and waiting to see how the users will behave.